c++/src/kj/time.c++

```diff
@@ -268,6 +268,9 @@ CappedArray<char, sizeof(int64_t) * 3 + 2 + 4> KJ_STRINGIFY(Date d) {
   return kj::toCharSequence(d - UNIX_EPOCH);
 }
 CappedArray<char, sizeof(int64_t) * 3 + 2 + 4> KJ_STRINGIFY(Duration d) {
+  bool negative = d < 0 * kj::SECONDS;
+  if (negative) d = 0 * kj::SECONDS - d;
```

Comment:
Oh, I guess the negative side of the signed integer range is slightly larger than the positive one, so this might still format one value weird?

Reply:
No, you were right the first time -- the most-negative value is stringified incorrectly!

Reply:
:open_mouth: